### DISCUSSION TITLE
Is it possible to use Denoflare to run Fresh projects?

### DISCUSSION THREAD

**Discussion by alew3** [skymethod/denoflare#45] [General]
Can I convert a Fresh project to run on Cloudflare?

**Comment by johnspurlock-skymethod** [comment]
I haven't tried it yet - though it sounds like fun.  Doesn't fresh use wasm esbuild on the server?  That's definitely possible on CF (I do it in https://esb.denoflare.dev for example), but it's not exactly fast.
